DEBUG    mypy.lib.proxy_pool.proxy_pool_helper:proxy_pool_helper.py:28 🔵 {'anonymous': '', 'check_count': 8, 'fail_count': 0, 'https': False, 'last_status': True, 'last_time': '2024-07-05 23:09:30', 'proxy': '183.234.215.11:8443', 'region': '中国 广东 深圳 移动', 'source': 'freeProxy11'}
WARNING  mypy.lib.proxy_pool.proxy_pool_helper:proxy_pool_helper.py:49 🟠 未找到可用的 (https)深圳 代理地址